assignments/assignment02.rb

```diff
@@ -0,0 +1,187 @@
+# ========================================================================================
+#  Assignment submitted by Gordon Turibamwe
+#
+# ========================================================================================
+# Assignment 2
+# ========================================================================================
+
+# ========================================================================================
+#  Problem 1 - `to_sentence`
+
+# implement method `to_sentence`
+
+# creates an english string from array
+
+def to_sentence(ary)  
+  # PSEUDO CODE
```

Comment:
Instead of thinking about iterating over the elements in the array, think about operating on the whole array at a time and using join to combine the elements into a string.